dataactcore/utils/report.py

```diff
@@ -4,25 +4,6 @@
 from dataactcore.models.lookups import FILE_TYPE
 
 
-def get_report_path(job, report_type):
-    """
-    Return the filename for the error report.
-    Does not include the folder to avoid conflicting with the S3 getSignedUrl method.
-    """
-    return report_file_name(job.submission_id, report_type == 'warning',
-                            job.file_type.name)
-
-
-def get_cross_report_name(submission_id, source_file, target_file):
-    """Return filename for a cross file error report."""
-    return report_file_name(submission_id, False, source_file, target_file)
-
-
-def get_cross_warning_report_name(submission_id, source_file, target_file):
-    """Return filename for a cross file warning report."""
-    return report_file_name(submission_id, True, source_file, target_file)
-
-
 def report_file_name(submission_id, warning, file_type, cross_type=None):
```

Comment:
Exactly; I renamed to use the same interface for normal file reports and cross file reports.